core/service/manager.go

```diff
@@ -21,6 +21,8 @@ import (
 	"encoding/json"
 	"errors"
 
+	"github.com/mysteriumnetwork/node/websocket"
```

Comment:
Websockets are for exposing external api like rest, it should not leak to other components. We are using internal event publisher/subscriber for events. Websockets should subscribe to interested events and broadcast them to outside world only.